crates/bevy_ui/src/flex/mod.rs

```diff
@@ -14,32 +14,32 @@ use bevy_transform::components::Transform;
 use bevy_utils::HashMap;
 use bevy_window::{Window, WindowId, WindowScaleFactorChanged, Windows};
 use std::fmt;
-use stretch::{number::Number, Stretch};
+use taffy::{number::Number, Taffy};
 
 pub struct FlexSurface {
-    entity_to_stretch: HashMap<Entity, stretch::node::Node>,
-    window_nodes: HashMap<WindowId, stretch::node::Node>,
-    stretch: Stretch,
+    entity_to_sprawl: HashMap<Entity, taffy::node::Node>,
+    window_nodes: HashMap<WindowId, taffy::node::Node>,
+    sprawl: Taffy,
 }
 
-// SAFE: as long as MeasureFunc is Send + Sync. https://github.com/vislyhq/stretch/issues/69
+// SAFE: as long as MeasureFunc is Send + Sync. https://github.com/vislyhq/sprawl/issues/69
```

Comment:
```suggestion
// SAFE: as long as MeasureFunc is Send + Sync. https://github.com/vislyhq/stretch/issues/69
```
This should refer to the old issue in the original `stretch` repo

Reply:
Or maybe we rather should create a copy of the issue in Taffy and link to it there? @alice-i-cecile

Reply:
Oh right. IMO we should just remove MeasureFunc here: https://github.com/DioxusLabs/taffy/issues/101

Reply:
What should I do for this PR?

Reply:
Create a new issue on Taffy and link to that.

Reply:
Done https://github.com/DioxusLabs/taffy/issues/146